genrl/environments/torch.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any
+
+import gym
+import torch
+from gym.core import Wrapper
+
+
+class TorchWrapper(Wrapper):
```

Comment:
Would be better if you inherited from `genrl.environments.GymWrapper` instead. That is sort of our BaseWrapper class rn. You can remove the entire `obs_shape` and `action_shape` part then.

Reply:
Yeah makes sense. I'll change that.